sdk/python/feast/infra/registry/sql.py

```diff
@@ -992,7 +992,7 @@ def _list_objects(
                 for row in rows:
                     proto = proto_class.FromString(row[proto_field_name])
                     if python_class == StreamFeatureView:
-                        obj = python_class.from_proto(proto, True if kwargs["skip_udfs"] else False)
+                        obj = python_class.from_proto(proto, kwargs.get("skip_udfs", True))
```

Comment:
I think the default value should actually be False? If `skip_udfs` isn't passed to this function, then this function was not called from the `feast apply` workflow, hence udfs shouldn't be skipped?

Reply:
Right, the default behavior should be to load the udfs